programs/uxd/src/state/credix_lp_depository.rs

```diff
@@ -71,8 +72,9 @@ pub struct CredixLpDepository {
     pub minting_fee_total_accrued: u128,
     pub redeeming_fee_total_accrued: u128,
 
-    // Collection of profit for the treasury
+    // Collection of the depository's profits
     pub profits_total_collected: u128,
+    pub profits_beneficiary_collateral: Pubkey,
```

Comment:
Nit: we could rename all accounts some day, this should be "profits_beneficiary_collateral_token_account", but not in the scope of this PR, and this fits the current nomenclature (which is wrong, just want to highlight)

Reply:
I see, yeah i kinda just rolled with the existing logic of:

```tsx
magicmoneyMint    // token mint account standard name
vincentMagicmoney // token account standard name
```

Reply:
resolving conversation before merge 👌